js/src/structured-data-blocks/faq/components/Question.js

```diff
@@ -0,0 +1,230 @@
+import PropTypes from "prop-types";
+
+const { Component } = window.wp.element;
+const { __ } = window.wp.i18n;
+const { IconButton } = window.wp.components;
+const { RichText, MediaUpload } = window.wp.editor;
+
+/**
+ * A Question and answer pair within a FAQ block.
+ */
+export default class Question extends Component {
+
+	/**
+	 * Constructs a Question editor component.
+	 *
+	 * @param {Object} props This component's properties.
+	 *
+	 * @returns {void}
+	 */
+	constructor( props ) {
```

Comment:
Maybe I am wrong, but I think this isn't necessary because you are only calling the super.